### PR TITLE
gelu support

### PR DESCRIPTION
**Pull request by yuchengliu1**
## Type of Change

feature
NO API changed 

## Description

add ffn gelu fusion to falcon and mpt
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/ITREX-cpp-graph-extension/109/

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp [hunk 1]
```diff
@@ -310,7 +310,34 @@ void jblas_fusion_FFN_SiLu_f32f32_forward(float* activation, void* w1ptr, void*
...
 bool jblas_fusion_FFN_GeLu_f32f32_support(void* w1ptr, void* w2ptr, int seq, int fin, int fmid, int fout) {
-  return false;
+  auto w1tmp = prologue::weight_comp::gemm_kblcok::PackedWeightParser::deserialBuffer(w1ptr, 0);
+  auto w2tmp = prologue::weight_comp::gemm_kblcok::PackedWeightParser::deserialBuffer(w2ptr, 0);
+  bool support = false;
+  if (w1tmp != nullptr && w2tmp != nullptr) {
+    prologue::PackedWeight* tmps[2] = {w1tmp, w2tmp};
+    auto sameKernel = samePackedWeight(tmps, 2);
+    if (sameKernel) {
+      if (sameKernel) {
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp:320]: duplicate?

Comment (yuchengliu1) [reply intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp:320]: fixed it.

**Comment by kevinintel** [comment]
please update next token latency

**Review by airMeng — APPROVED** [review]

**Comment by yuchengliu1** [comment]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/yucheng3/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/yucheng3/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">



Model | Input | Output | Batchsize | Cores/Instance | Precision | Eval Time per Token | Memory | 1st Latency | Total Time | P90 Latency Time | P99 Latency Time
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
mpt-7b | 32 | 32 | 1 | 56 | q4_j_b128 | 23.67 | 5616.94 | 81.53 | 815.43 | 23.93 | 81.53
mpt-7b | 1024 | 32 | 1 | 56 | q4_j_b128 | 33.35 | 5551.02 | 15008.73 | 16042.63 | 33.53 | 15008.73
falcon-7b | 32 | 32 | 1 | 56 | q4_j_b128 | 25.08 | 5418.33 | 74.11 | 851.5 | 25.31 | 74.11
falcon-7b | 1024 | 32 | 1 | 56 | q4_j_b128 | 27.99 | 5376.73 | 4793.91 | 5661.45 | 28.29 | 4793.91



</body>

</html>

**Comment by airMeng** [comment]
> 128	27.99

can you list the comparison before & after?

**Pull request merged**